docs/source/index.mdx

```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
     Learn more
   </Button>
 </div>
+
+## Recommended workouts
+
+<MenuWrapper>
+
+<WorkoutMenuItem
+  icon={HeadLight}
+  keyWord="Discover"
+  otherWords="Apollo"
+>
+
+* [The Apollo GraphQL platform](./intro/platform/)
+* [Why GraphQL?](./intro/benefits/)
+* [Introduction to Apollo Server](https://www.apollographql.com/docs/apollo-server/)
+* [Introduction to Apollo Client](https://www.apollographql.com/docs/react/)
+* [Overview of Apollo Graph Manager](https://www.apollographql.com/docs/graph-manager/)
```

Comment:
Could I suggest a section on "Getting set up with Apollo tooling" which introduces users to the chrome extension, the CLI, and VS Code extension?

Reply:
That's currently under "Dive in". I'll move it up a bit.